src/reference/02-DetailTopics/05-Plugins-and-Best-Practices/02-Plugins.md

```diff
@@ -508,6 +508,29 @@ libraryDependencies ++=
         Seq.empty
 ```
 
+### Publishing a plugin
+
+Sbt plugins can be published as any other projects. 
+
+However, there is one caveat if you attempt to publish your plugin to a repository that follows the Maven scheme.
```

Comment:
```suggestion
However, there is one caveat if you attempt to publish your plugin to a repository that follows the Maven naming scheme.
```

Reply:
I usually say "Maven layout" (as opposed to "Ivy layout").